int32 intVal
---
int32 retVal
